packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -64,6 +64,7 @@ class UrlLauncherPlugin extends UrlLauncherPlatform {
     // See https://github.com/flutter/flutter/issues/51461 for reference.
     final String target = webOnlyWindowName ??
         ((_isSafari && _isSafariTargetTopScheme(url)) ? '_top' : '');
+    // ignore: unsafe_html
     return _window.open(url, target);
```

Comment:
Good callout, this one probably needs a `noopener` (at the very least) in the open call, similar to what we do here: https://github.com/flutter/plugins/blob/main/packages/url_launcher/url_launcher_web/lib/src/link.dart#L181